### PR TITLE
Update dependencies, deduped rules, fixed plugins

### PR DESCRIPTION
**Pull request by JustFly1984**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,16 +24,6 @@ module.exports = {
     'no-console': 0,
     'no-alert': 0,
     'no-nested-ternary': 0,
-    'comma-dangle': [
```
Comment (zsherman) [inline index.js]: did you remove these intentionally?

**Comment by JustFly1984** [comment]
it is duplicated, I just get rid of last one

2018-03-24 2:48 GMT+08:00 Zach Sherman <notifications@github.com>:

**Comment by JustFly1984** [comment]
BTW coma-dangle cause error in firefox. In my humble opinion it should not
exist, so I override this rule in my own eslintrc anyway

2018-03-24 2:59 GMT+08:00 Alexey Lyahov <justfly1984@gmail.com>:

**Comment by zsherman** [comment]
Gotcha, I'm not a huge fan of it myself but @tannerlinsley has a strong preference for it I believe.

**Comment by JustFly1984** [comment]
eslint's auto-fix on save solves all my issues with coma-dangle and
everything else. There is to many opinionated stuff in JS, and I'm too old
for holy-wars.

2018-03-24 3:03 GMT+08:00 Zach Sherman <notifications@github.com>:

**Comment by tannerlinsley** [comment]
Comma dangle should stay. Override locally if you prefer otherwise.

**Comment by JustFly1984** [comment]
I'm already overriding locally. There was 2 duplicated rules `coma-dangle`,
I get rid of last one.

2018-03-24 3:19 GMT+08:00 Tanner Linsley <notifications@github.com>:

**Comment by tannerlinsley** [comment]
Published new version 👍

**Comment by JustFly1984** [comment]
Thank you!

2018-03-24 4:59 GMT+08:00 Tanner Linsley <notifications@github.com>:

**Pull request merged**